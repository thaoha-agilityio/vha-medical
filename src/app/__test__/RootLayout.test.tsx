import { render } from '@testing-library/react';
import RootLayout from '../layout';

jest.mock('next-themes', () => ({
  ThemeProvider: jest.fn(),
}));

describe('NotFound page test case', () => {
  const setup = (
    props: Readonly<{
      children: React.ReactNode;
    }>,
  ) => render(<RootLayout {...props} />);

  it('should render correctly', () => {
    const { asFragment } = setup({ children: <div>mock children</div> });

    expect(asFragment()).toMatchSnapshot();
  });
});
