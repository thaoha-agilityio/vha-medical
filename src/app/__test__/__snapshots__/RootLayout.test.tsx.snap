// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound page test case should render correctly 1`] = `
<DocumentFragment>
  <div
    data-overlay-container="true"
  />
</DocumentFragment>
`;
